table/options.go

```diff
@@ -249,3 +250,17 @@ func (m Model) WithColumns(columns []Column) Model {
 
 	return m
 }
+
+// WithFilterInput sets the filter input for the table.  This can be used as API of filter input.
```

Comment:
```suggestion
// WithFilterInput makes the table use the provided text input bubble for
// filtering rather than using the built-in default.  This allows for external
// text input controls to be used.
```

I want to clarify the documentation a bit here, but open for discussion.